.github/workflows/trivy.yml

```diff
@@ -0,0 +1,29 @@
+name: trivy
+
+on:
+  pull_request:
```

Comment:
do we want this to run on merge to main too?

Reply:
Good catch, updated it to run also on a push to main